src/dropdowns/dropdown-trigger.ts

```diff
@@ -0,0 +1,46 @@
+import {Directive, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownTrigger]',
+  host: {
+    'aria-haspopup': 'true',
+    'tabindex': '0',
```

Comment:
@sadesyllas No need manually add `tabindex`. The best practice is that trigger elements are focusable by default, usually a `<button>` or an `<a>`. We shouldn't care for people not following best practices. Let's help make the world better :)